External/GA/ga/BinaryStringChromosomePolicy.cpp

```diff
@@ -0,0 +1,26 @@
+/* 
```

Comment:
As a logistics point: these files should all have a standard copyright at the top so that it's clear they're part of the RDKit. Something like:
```
//
//  Copyright (C) 2020 Gareth Jones
//
//   @@ All Rights Reserved @@
//  This file is part of the RDKit.
//  The contents are covered by the terms of the BSD license
//  which is included in the file license.txt, found at the root
//  of the RDKit source tree.
//
```

Reply:
I'll add the headers

Reply:
Since I'm doing this as part of Glysade is 
```
//  Copyright (C) 2020 Gareth Jones, Glysade LLC
```
ok?

Reply:
Indeed!  I assigned mine to Novartis back in the day.

Reply:
> Since I'm doing this as part of Glysade is
> 
> ```
> //  Copyright (C) 2020 Gareth Jones, Glysade LLC
> ```
> 
> ok?

Of course. I was just making a suggestion

Reply:
All new files have the standard header.